modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/system/CPDefinitionSystemObjectDefinitionMetadata.java

```diff
@@ -147,10 +163,75 @@ public int getVersion() {
 		return 2;
 	}
 
+	@Override
+	public void updateBaseModel(
+			long primaryKey, User user, Map<String, Object> values)
+		throws Exception {
+
+		ProductResource productResource = _getProductResource(user);
+
+		CPDefinition cpDefinition = _cpDefinitionLocalService.getCPDefinition(
+			primaryKey);
+
+		productResource.patchProduct(
+			cpDefinition.getCProductId(), _getProduct(values));
+	}
+
+	private Map<String, String> _getLanguageIdMap(
```

Comment:
Implement this method like protected in the base class :+1:

Reply:
Done! I added a separate commit to simplify new review.
Thanks!